src/scripts/main.js

```diff
@@ -1,5 +1,51 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const wall = document.querySelector('.wall');
+
+wall.addEventListener('click', e => {
```

Comment:
Hmm, your code works fine, but think we can just simplify this code and reduce it. What about just this solution? 

```suggestion
wall.addEventListener('click', e => {
  const spider = document.querySelector('.spider');

  spider.style.top = `${e.offsetY}px`;
  spider.style.left = `${e.offsetX}px`;
});

```